graviti/dataframe/row/series.py

```diff
@@ -5,10 +5,12 @@
 
 """The implementation of the Graviti Series."""
 
-from typing import Any, Dict, Iterable, List, Optional, Union, overload
+from typing import Any, Dict, Iterable, List, Optional, Union, overload, Tuple
 
 from graviti.dataframe.series import SeriesBase
 
+_INDENT = " "
+_MAX_REPE_NUMBER = 10
```

Comment:
why we have `_INDENT` and `_MAX_REPR_NUMBER` in two files?
can we reuse them?
also the `_MAX_REPE_NUMBER` is spelt wrong